user_preferences_bindings_wasm/package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@xmtp/ecies-bindings-wasm",
+  "name": "@xmtp/user_preferences-bindings-wasm",
```

Comment:
nit: seems weird to use snake_case and kebab-case together

can we choose 1 or the other?

Reply:
Good catch. We should make all kebab for the package.json name.

Reply:
Oops that was my bad! Nice catch.